src/ProjectFileTools.MSBuild/WorkspaceManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ProjectFileTools.MSBuild
+{
+    /// <summary>
+    /// Provides the correct workspace
+    /// </summary>
+    public class WorkspaceManager
```

Comment:
Consider extracting this into an interface and export the interface instead of the implementation. This will allow for mocking in unit tests.